### ISSUE TITLE
Launcher does not start

### ISSUE THREAD

**Issue by PerfectLogicsLab** [remyroy/CDDA-Game-Launcher#273]
* CDDA Game Launcher version: 1.3.18
* OS: Windows-7-6.1.7601-SP1 (64-bit)
* Type:  `<class 'TypeError'>`
* Value: '<' not supported between instances of 'NoneType' and 'str'
* Traceback:
```
  File "cddagl\ui.py", line 3169, in lb_http_finished

```

**Comment by PerfectLogicsLab** [comment]
OS: Windows-7-6.1.7601-SP1 (64-bit)
Type: <class 'TypeError'>
Value: '<' not supported between instances of 'NoneType' and 'str'
Traceback:
 File "cddagl\ui.py", line 3169, in lb_http_finished

**Comment by lool228** [comment]
Описание: [Напишите, что вы делали и что произошло]
* Версия: 1.3.18
* OS: Windows-10-10.0.17134 (64-bit)
* Тип: `<class 'TypeError'>`
* Значение: '<' not supported between instances of 'NoneType' and 'str'
* Диагностика:
```
  File "cddagl\ui.py", line 3169, in lb_http_finished

```

**Comment by remyroy** [comment]
This is a duplicate of #271 and should be fixed with 1.3.19 .
